docker/xgo-image-deb6/beats-builder/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM tudorg/xgo-deb6-1.4.2
```

Comment:
I'm currently using my personal account indeed.
I'm hoping we will have an internal docker registry at some point for these kind of things